sdk/tests/deqp/modules/shared/glsShaderLibrary.js

```diff
@@ -145,9 +145,7 @@ var gluShaderUtil = framework.opengl.gluShaderUtil;
         if (trimFront)
             str = str.replace(/^\s*\n/, '');
         return str
-            .replace('\\n', '\n')
-            .replace('\\t', '\t')
-            .replace(/\\/g, '');
+            .replace(/\\\\/g, '\\');
```

Comment:
See C++ code:
https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/opengl/gluShaderLibrary.cpp#537

tow \ is converted to one \: "\\\\" -> "\\", e.g. "\\note" should be converted to "\note" other than
 "
ote".